src/components/CollectionDropdown/CollectionDropdown.tsx

```diff
@@ -6,9 +6,9 @@ import './CollectionDropdown.css'
 
 export default class CollectionDropdown extends React.PureComponent<Props> {
   componentDidMount() {
-    const { address, onFetchCollections } = this.props
+    const { address, isCollectionPublished, onFetchCollections } = this.props
```

Comment:
What do you think if we pass an entire object of the possible params for the fetch? it could be called `fetchCollectionParam`
```suggestion
    const { address, onFetchCollections, fetchCollectionParams } = this.props
```

Reply:
Nice proposal! doing it now 🚀